### PR TITLE
Updated the URL for Okta

### PR DESCRIPTION
**Pull request by mmahadevan-okta**

### CODE REVIEW

File: _data/identity.yml [hunk 1]
```diff
@@ -100,8 +100,9 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
+      phone: Yes
       hardware: Yes
-      doc: http://www.okta.com/what-we-do/multifactor-authentication.html
+      doc: https://www.okta.com/products/adaptive-multi-factor-authentication/
```
Comment (mxxcon) [inline _data/identity.yml]: `doc:` url should be not a marketing product page with what okta is selling, but a link to a documentation how a user can enable 2fa for their okta account.

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Review by mmahadevan-okta — COMMENTED** [review]
Updated the URL to the support page.

**Comment by mmahadevan-okta** [comment]
Changed URL to https://help.okta.com/en/prod/Content/Topics/Security/MFA.htm

**Review by stephengroat — APPROVED** [review]

**Review by jamcat22 — APPROVED** [review]
:+1:

**Review by mxxcon — COMMENTED** [review]
If the current URL is as good as it gets 👍, otherwise a link to the actual _instructions_ would be nice.

**Review by mxxcon — APPROVED** [review]

**Pull request merged**